GitUI/SpellChecker/IWordAtCursorExtractor.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace GitUI.SpellChecker
+{
+    internal interface IWordAtCursorExtractor
```

Comment:
since there is only one implementation you can keep the interface in the same file with the implementation